scripts/zones/Horlais_Peak/mobs/Chlevnik.lua

```diff
@@ -13,70 +13,59 @@ entity.onMobInitialize = function(mob)
     mob:addImmunity(xi.immunity.PETRIFY)
     mob:addImmunity(xi.immunity.SILENCE)
     mob:setBehavior(bit.bor(mob:getBehavior(), xi.behavior.NO_TURN))
+    mob:addListener('WEAPONSKILL_STATE_EXIT', 'FINAL_METEOR_DEATH', function(mobArg, skillID)
+        if skillID == xi.mobSkill.FINAL_METEOR then
+            mobArg:timer(7000, function(mobArgTimer)
+                mobArgTimer:setUnkillable(false)
+                mobArgTimer:setHP(0)
+            end)
+        end
+    end)
 end
 
 entity.onMobSpawn = function(mob)
     mob:setAnimationSub(3)
     mob:setUnkillable(true)
+    mob:setAutoAttackEnabled(true)
+    mob:setMagicCastingEnabled(true)
+    mob:setMobAbilityEnabled(true)
     mob:setMod(xi.mod.TRIPLE_ATTACK, 5)
     mob:addMod(xi.mod.STUNRES, 90)
     mob:setMobMod(xi.mobMod.NO_MOVE, 0)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
-    mob:setTP(3000) -- opens fight with a skill
-    mob:setUnkillable(true)
-    mob:setLocalVar('Meteor', 0)
+    mob:setMod(xi.mod.REGAIN, 50)
+    mob:setTP(3000)
+    mob:setLocalVar('finalMeteor', 0)
 end
 
 entity.onMobEngage = function(mob, target)
-    mob:setLocalVar('delay', GetSystemTime() + 30)
+    mob:setLocalVar('meteorDelay', GetSystemTime() + 30)
 end
 
 entity.onMobFight = function(mob, target)
-    if mob:getHPP() > 25 then
-        mob:setMod(xi.mod.REGAIN, 80)
-    else
-        mob:setMod(xi.mod.REGAIN, 50)
-    end
-
-    local delay = mob:getLocalVar('delay')
+    local delay = mob:getLocalVar('meteorDelay')
     if GetSystemTime() > delay then -- Cooldown on Meteor is 30 seconds.
         mob:castSpell(xi.magic.spell.METEOR, target)
-        mob:setLocalVar('delay', GetSystemTime() + 30)
+        mob:setLocalVar('meteorDelay', GetSystemTime() + 30)
     end
 
-    if mob:getHP() == 1 and mob:getLocalVar('Meteor') == 0 then
+    if mob:getHP() == 1 and mob:getLocalVar('finalMeteor') == 0 then
         mob:setAutoAttackEnabled(false)
         mob:setMagicCastingEnabled(false)
         mob:setMobAbilityEnabled(false)
-        mob:setLocalVar('Meteor', 1)
+        mob:setMobMod(xi.mobMod.NO_MOVE, 1)
+        mob:setAnimationSub(1)
```

Comment:
Doesnt this animation make it fall?
It should try to use the skill and then fall, right?